### DISCUSSION TITLE
Can't navigate through pages in the Pull Requests tab of any repo

### DISCUSSION THREAD

**Discussion by zac-wilson-asecca** [community/community#129627] [Pull Requests]
### Select Topic Area

Bug

### Body

Attempting to click the page numbers or next page button at the bottom of the list of pull requests for any repo causes the first page of results to load rather than the requested page. This doesn't affect the Issues tab, only the Pull Requests tab. This happens regardless of whether I'm logged in or not, and happens on every repo I've tried to look at today.

**Comment by lukasz-jarocki-sonarsource** [comment]
I am facing the same issue.

Steps to reproduce:
1. Go to closed pull requests in any repository, in example: https://github.com/SonarSource/sonarqube/pulls?q=is%3Apr+is%3Aclosed
2. Scroll down to the bottom of the page and click "2"nd page.

The page just refreshes.

**Comment by Gudahtt** [comment]
I've been experiencing the same thing. I was able to work around it by manually adding `?page=[page number]` (e.g. `?page=2`) to the URL.

Edit: To clarify, I also removed any pre-existing query parameters.

**Comment by Yay295** [comment]
It looks like `.../issues?page=2&pulls_only=true&q=is%3Apr` is being redirected to `.../pulls?q=is%3Apr`. That is, `issues?pulls_only=true` becomes `pulls`, but the `page` parameter is lost in the transition.

**Comment by saikris12** [comment]
Facing the same issue. Adding `?page=2` also does not work either.

**Comment by D4ndellion** [comment]
Seems fixed now
